src/pip/_internal/basecommand.py

```diff
@@ -200,7 +200,7 @@ def main(self, args):
 
         if sys.version_info[:2] == (3, 3):
             warnings.warn(
-                "Python 3.3 supported has been deprecated and support for it "
+                "Python 3.3 support has been deprecated and support for it "
```

Comment:
The second `support` could be dropped: `Python 3.3 support has been deprecated and will be removed in the future [...]`.